test/tools/d_do_test.d

```diff
@@ -16,6 +16,8 @@ import std.stdio;
 import std.string;
 import core.sys.posix.sys.wait;
 
+//version = MeasureTime;
```

Comment:
Maybe use an environment variable, s.t. this is easier to enable and the code doesn't rot (i.e. is at least checked by the compiler).